src/patterns.rst

```diff
@@ -224,9 +225,6 @@ Literal Patterns
 :dp:`fls_pah15qa54irs`
 A :t:`literal pattern` is a :t:`pattern` that matches a :t:`literal`.
 
-:dp:`fls_v7iv7x9gy9qm`
-A :t:`literal pattern` is always :t:`refutable`.
```

Comment:
Isn't this important? I see there is a blanket "if it is not irrefutable, then it is refutable" below, but it is somewhat too late.

Reply:
We only ever seem to specify when a pattern is irrefutable in the other cases, so I removed this for uniformity. Alternatively, we could move the refutability chapter up to the front which would align with how we do things with expressions

Reply:
That's a good idea, let's do that!

Reply:
Actually it might be better to move the text of 5.1. Patterns to 5. Patterns, and make 5.3. Refutability into 5.1. This way we present the concept of a "pattern" and then we can talk about its refutability.